### PR TITLE
add option to disable segment

### LINKED ISSUE #27: Add option to disable segment
I have multiple environments and I would like to be able to disable segment from the config for some of the envs.

Something like:

```
  config :segment, enable: false
```

### PR DESCRIPTION
**Pull request by steffenix**
closes #27

### CODE REVIEW

File: lib/segment/analytics.ex [hunk 1]
```diff
@@ -164,6 +164,6 @@ defmodule Segment.Analytics do
 
   @spec call(Segment.segment_event()) :: :ok
   def call(event) do
-    Segment.Config.service().call(event)
+    if Segment.Config.enabled?, do: Segment.Config.service().call(event)
```
Comment (gnimona) [inline lib/segment/analytics.ex:167]: else return `:ok` so to follow the spec?

Comment (chakming) [inline lib/segment/analytics.ex:167]: else return `:ok` so to follow the spec?

```suggestion
  case Segment.Config.enabled? do
    true -> Segment.Config.service().call(event)
    _ -> :ok
  end
```

Comment (steffenix) [reply lib/segment/analytics.ex]: I prefer a simple else.

**Comment by kwayebopp** [comment]
Isn't this functionality permitted already via `config :segment, send_to_http: false`?

From the docs:

config :segment, :send_to_http 

If set to false, the libray will override the Tesla Adapter implementation to only log segment calls to debug but not make any actual API calls. This can be useful if you want to switch off Segment for test or dev. Default value is true.

**Comment by steffenix** [comment]
ok that should be enough for my staging envs.

**Pull request closed without merging**